core/drivers/crypto/se050/core/ecc.c

```diff
@@ -212,6 +212,9 @@ static TEE_Result se050_inject_public_key(sss_se05x_object_t *k_object,
 	if (st != kStatus_SSS_Success)
 		return TEE_ERROR_BAD_PARAMETERS;
 
+	(void)sss_se05x_key_store_create_curve(&se050_session->s_ctx,
```

Comment:
What happens if this function fails?

Reply:
if the curve doesnt exist in the SE, the next function will fail to inject the key.
if the curve already exist in the SE and this function for whatever reason fails, it all will work as it should.

Apparently some versions of the API used to return an error if the curve was already in the SE (which would be a pass for what we need here) hence why I coded it this way. Not a big deal I dont think.

Reply:
Makes sense. Please drop the `(void)` and add a comment along what you just said.

Reply:
@jenswi-linaro pushed